ctre/basemotorcontroller.py

```diff
@@ -93,18 +96,40 @@ def set(self, mode: ControlMode, demand0: float, demand1: float = 0.0):
             In :attr:`.ControlMode.Position` mode, output value is in encoder ticks or an analog value, depending on the sensor.
             
             In :attr:`.ControlMode.Follower` mode, the output value is the integer device ID of the talon to duplicate.
-        :type demand0: float
         :param demand1:
             Supplemental value.  This will also be control mode specific for future features.
-        :type demand1: float
-
-        see :meth:`.selectProfileSlot` to choose between the two sets of gains.
+        :param demand1Type:
+            The demand type for demand1.
+            
+            * Neutral: Ignore demand1 and apply no change to the demand0 output.
+            * AuxPID: Use demand1 to set the target for the auxiliary PID 1.
+            * ArbitraryFeedForward: Use demand1 as an arbitrary additive value to the
+              demand0 output.  In PercentOutput the demand0 output is the motor output,
+              and in closed-loop modes the demand0 output is the output of PID0.
+    
+        Arcade Drive Example::
+        
+            _talonLeft.set(ControlMode.PercentOutput, joyForward, DemandType.ArbitraryFeedForward, +joyTurn)
+            _talonRght.set(ControlMode.PercentOutput, joyForward, DemandType.ArbitraryFeedForward, -joyTurn)
+                    
+        Drive Straight Example::
+                            
+            # Note: Selected Sensor Configuration is necessary for both PID0 and PID1.
+            _talonLeft.follow(_talonRght, FollowerType.AuxOutput1)
+            _talonRght.set(ControlMode.PercentOutput, joyForward, DemandType.AuxPID, desiredRobotHeading)
+                            
+        Drive Straight to a Distance Example::
+        
+            # Note: Other configurations (sensor selection, PID gains, etc.) need to be set.
+            _talonLeft.follow(_talonRght, FollowerType.AuxOutput1);
+            _talonRght.set(ControlMode.MotionMagic, targetDistance, DemandType.AuxPID, desiredRobotHeading)
+        
         """
         self.controlMode = mode
         self.sendMode = mode
 
         if self.controlMode == ControlMode.PercentOutput:
```

Comment:
Hm. I just realised, shouldn't these comparisons be identity checks (`is`) instead?

Reply:
I think `==` is fine.

Reply:
Well, realistically we don't actually support passing ints here, so calling `__eq__` seems wasteful.